app/components/shared/dashboard_component/dashboard_component.html.slim

```diff
@@ -0,0 +1,16 @@
+.dashboard__header
+  .govuk-width-container
+    h1.govuk-heading-m class="govuk-!-margin-bottom-2"
+      = @heading
+
+  .govuk-width-container
+    - if @link
+      .filters-information.moj-action-bar
+        = link_to t("buttons.create_job"), @link, method: :post, class: "govuk-button govuk-!-margin-bottom-0 govuk-!-margin-top-0 dashboard__link"
```

Comment:
This has changed in https://github.com/DFE-Digital/teaching-vacancies/pull/2598

Reply:
Also, if we're passing in the link, should we pass in the link text too so it isn't tied to `t("buttons.create_job")`?